exercises/code-scanning.md

```diff
@@ -162,36 +162,67 @@ By creating our own [code scanning suite](https://codeql.github.com/docs/codeql-
         - <insert rule id of false positive>
     ```
 
-2. Configure the file `.github/codeql/codeql-config.yml` to use our suite
+2. Configure the file `.github/codeql/codeql-config.yml` to use our suite.
 
-    **Hint** We are now running both the default code scanning suite and our own custom suite.
-    To prevent double results disable the default queries with the option `disable-default-queries: true`
+    **Hint**: We are now running both the default code scanning suite and our own custom suite.
+    To prevent CodeQL from resolving queries twice, disable the default queries with the option `disable-default-queries: true`
+    
+<details>
+<summary>Solution</summary>
+
+```yaml
+name: "My CodeQL config"
+
+disable-default-queries: true
+
+queries:
+    - uses: ./custom-queries/code-scanning.qls
+```
+</details>
 
 3. After the code scanning action has completed, is the false positive still there?
 
+4. Try running additional queries with `security-extended` or `security-and-quality`. What kind of results do you see?
+
+**Note**: If you want to use these additional query suites and the custom query suite you've made, make sure to import the proper query packs to continue to exclude certain queries.
+
 <details>
 <summary>Solution</summary>
 
 ```yaml
 # Reusing existing QL Pack
-- import: codeql-suites/javascript-code-scanning.qls
+- import: codeql-suites/javascript-security-and-quality.qls
```

Comment:
I think this change should also be made on line 152 ect as well.

Reply:
Oooh let's retain it actually since that at point of the exercise we haven't dealt with using additional queries

Reply:
Just added those 2 extra spaces! Good catch!

Reply:
ahh right right ! my bad!

ok cool everything lgtm 👍